docs/misc/message_system.md

```diff
@@ -0,0 +1,242 @@
+                        ],
+                        "linux": "<20.04",
+                        "windows": "!",
+                        "android": "*",
+                        "ios": "13"
+                    },
+                    "environment": {
+                        "desktop": "<21.5",
+                        "mobile": "!",
+                        "web": "<22"
+                    },
+                    "browser": {
+                        "firefox": [
+                            "82",
+                            "83"
+                        ],
+                        "chrome": "*",
+                        "chromium": "!"
+                    },
+                    "transport": {
+                        "bridge": [
+                            "2.0.30",
+                            "2.0.27"
+                        ],
+                        "webusbplugin": "*"
+                    },
+                    /*
+                    - If key is not available (undefined), then it can be whatever.
+                    - Currently supported keys are "tor" and coin symbols from "enabledNetworks".
+                    - The bitwise operation is OR.
+                    */
+                    "settings": [
+                        { 
+                            "tor": true,
+                            "btc": true
+                        },
+                        {
+                            "tor": false,
+                            "ltc": true
+                        }
+                    ],
+                    // Empty device array is targeting users without a connected device.
+                    "devices": [ 
+                        {
+                            // Possible values: "1" or "T"
+                            "model": "1", 
+                            "firmware": "1.9.4",
+                            "vendor": "trezor.io"
+                        }
+                    ]
+                }
+            ],
+            "message": {
+                // Used for remembering dismissed messages.
+                "id": "0f3ec64d-c3e4-4787-8106-162f3ac14c34",
+                /*
+                - Existing banners have defined priorities. 
+                - The range is 1 to 10.
+                */
+                "priority": 10,
+                // When user closes the message, it will never show again.
+                "dismissible": true,
+                /* 
+                Variants:
+                - info (blue)
+                - warning (orange)
+                - critical (red)
+                */
+                "variant": "warning",
+                // Options: banner, modal, context, super-context
+                "category": "banner",
+                /*
+                - Message shown to a user. 
+                - Current implementation uses only "en-GB".
+                */
+                "content": {
+                    "en-GB": "New Trezor firmware is available!",
```

Comment:
It is text-only by design?

Reply:
Yes, it is text-only. I do not think HTML makes sense here.

@tsusanka or does it?

Reply:
Hmm :thinking:. HTML no but I am wondering if markdown wouldn't be nice. But that's out of scope for now, let's ship it and iterate later :).